.github/workflows/token.yml

```
@@ -0,0 +1,29 @@
+name: 'figma-tokens style-dictionary actions'
+on:
+  pull_request:
+
+jobs:
+  generate_scss:
+    name: generate_scss
+    runs-on: ubuntu-latest
+    # 'figma' という名前を含むブランチのみ動くように設定
+    if: contains(github.head_ref, 'figma')
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+
+      - name: Install Dependencies
+        run: yarn install
```

```suggestion
        run: yarn install --frozen-lockfile
```

`yarn install`ではなく`yarn install --frozen-lockfile`を用いてください。

https://classic.yarnpkg.com/lang/en/docs/cli/install/#toc-yarn-install-frozen-lockfile